modules/security-context-constraints-rbac.adoc

```diff
@@ -54,13 +54,13 @@ the `resourceNames` field.
 <6> A list of verbs to apply to the role.
 
 A local or cluster role with such a rule allows the subjects that are
-bound to it with a RoleBinding or a ClusterRoleBinding to use the
+bound to it with a role binding or a cluster role binding to use the
 user-defined SCC called `scc-name`.
 
 [NOTE]
 ====
 Because RBAC is designed to prevent escalation, even project administrators
-will be unable to grant access to an SCC. By defualt, they are not
+are unable to grant access to an SCC. By default, they are not
```

Comment:
Does it make sense to elaborate on _they_ here? Is it RBACs or administrators?

Reply:
Not sure, so not changing with the scope of this PR being for formatting updates.